IronSoftware.Drawing/IronSoftware.Drawing.Common.ConsoleTest/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿// See https://aka.ms/new-console-template for more information
```

Comment:
Good that we have test!

Alternatively, I think we could get away by grabbing memory usage information from either `GC` or `System.Diagnostics`, and validate if it doesn't reach higher than what rough goal we set.